src/main/java/Todo.java

```diff
@@ -0,0 +1,21 @@
+public class Todo extends Task {
+    private Todo(String taskName, int taskId, boolean isCompleted) {
+        super(taskName, taskId, isCompleted);
+    }
```

Comment:
A line break might be suitable between the 2 constructors.

Reply:
Noted! :)